package.json

```diff
@@ -27,7 +27,7 @@
 		"@js-joda/core": "^5.4.2",
 		"@js-joda/locale_en": "^4.8.6",
 		"@js-joda/timezone": "^2.15.0",
-		"@lifesg/react-design-system": "^1.0.0-alpha.7",
+		"@lifesg/react-design-system": "^1.0.0-alpha.10",
```

Comment:
You might want to consider updating to the latest to test if it still works as expected

Reply:
ok I have updated it to alpha.14